scrnaseq.Rmd

```diff
@@ -1192,64 +1172,66 @@ We use Principal Component Analysis (PCA) to summarise a dataset, overcoming noi
 To decide how many PCs to include in downstream analyses, we visualise the cells and genes that define the PCA.
 ```{r pca}
 # Run PCA for default normalisation
-sc = Seurat::RunPCA(sc, features=Seurat::VariableFeatures(object=sc), verbose=FALSE)
+sc = Seurat::RunPCA(sc, features=Seurat::VariableFeatures(object=sc), verbose=FALSE, npcs=min(50, ncol(sc)))
```

Comment:
`ncol(sc)` is the number of cells?

Reply:
yes, and nrow is the number of features in the active assay